.storybook/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const config = require( '../webpack.config' );
+config.module.rules = [
+	...config.module.rules.filter( ( rule ) => ! rule.test.test( 'file.scss' ) ),
```

Comment:
We need to redefine the sass loader to avoid the ExtractTextPlugin, and the only way I've found to exclude it is to run the test with a dummy `file.scss`